docs/reference/technology/connectivity/connectivity.md

```diff
@@ -32,85 +32,83 @@ Our Socket API standardizes all of our connectivity options. It supports both IP
 
 ## Non-IP networking
 
 ### LoRaWAN
 
-LoRaWAN is optimized for low power consumption and a low memory footprint (allowing low-cost devices). The downside is a low throughput of no more than 50 kbps, as well as delays. It is a non-IP-based technology.
+LoRaWAN, a technology not based on IP, is optimized for low power consumption and a low memory footprint (allowing low-cost devices). The downside is a low throughput of no more than 50 kbps, as well as delays.
 
-Because of its long range (up to 20 km) and low power, it is suitable for low data rate, outdoors solutions.
+Because of its long range (up to 20 km) and low power, it is suitable for outdoor solutions with low data rates.
 
-The [LoRa](lora-tech.html) section and [LoRa tutorial](../tutorials/LoRa-tutorial.html) describe LoRA networking.
+The [LoRa](lora-tech.html) section and [LoRa tutorial](../tutorials/LoRa-tutorial.html) describe LoRA networking in detail.
 
 ### NFC
 
 <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/n_mark.png)<span>NFC</span></span>
 
-Near-field communication (NFC) is a short range (few centimeters) wireless technology standard for personal area networks. Typical uses of NFC are commissioning, easy sharing of small content and Bluetooth connection initiation/out-of-band pairing.
+Near-field communication (NFC) is a short range (few centimeters) wireless technology standard for personal area networks. Typical uses of NFC are commissioning, sharing of small content and Bluetooth connection initiation and out-of-band pairing.
 
 To learn how to use NFC with Mbed OS, please refer to the [Mbed OS NFC overview](../apis/nfc.html).
 
 ### NB-IoT cellular
 
-Non-IP Data Delivery (NIDD) is a new feature for communication over NB-IoT. It is enabled by Control Plane cellular IoT EPS optimization and meant to provide improved support of small data transfer. It does this by transporting user data over the control channel, thus reducing the total number of control plane messages when handling a short data transaction.
+Non-IP Data Delivery (NIDD) is a new feature for communication over NB-IoT. Control Plane cellular IoT EPS optimization enables NIDD, which is meant to provide improved support of small data transfer. It does this by transporting user data over the control channel, thus reducing the total number of control plane messages when handling a short data transaction.
 
-To learn how to use this feature with Mbed OS, please refer to [CellularNonIPSocket](../apis/non-ip-cellular-socket.html).
+To learn how to use this feature with Mbed OS, please refer to the [CellularNonIPSocket API reference](../apis/non-ip-cellular-socket.html).
 
 ### Memory needs for Pelion-connected devices
 
 Mbed OS's baseline memory footprint, without Pelion connectivity, is 2.8kb of RAM and 8.2kb of flash. When you add connectivity and full functionality, the footprint grows:
```

Comment:
Are these numbers going to change? What about the ones for Client and Client Lite?

Reply:
@AnotherButler someone should check the OS baseline numbers and define a bit what that means.

For Client, you should just link to Cloud docs: https://www.pelion.com/docs/device-management/current/cloud-requirements/requirements-for-production-devices.html which lists the up to date values. Numbers below are quite old. Client specific data here could be removed.

Reply:
Under header #Example memory figures (that does not seem to allow linking directly to that)

Reply:
@bulislaw Would you like to check the Mbed OS baseline memory numbers and expand on them, or would you rather I remove the specific numbers?

Reply:
I'd rather you remove them. I don't even know what does it mean "baseline memory", what features it includes (rtos? bare metal?) or which platfrom? 8K is probably excluding the required partner's SDK. It may be miss-leading for our users.